### PR TITLE
Minor fixups

### PR DESCRIPTION
**Pull request by luk1337**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -237,7 +239,9 @@
 
         // {% if args.encryption_client_side || args.encryption_server_side || args.enable_readonly %}
         if (passwordField.value.trim() != "") {
+            // {% if !args.no_file_upload %}
```
Comment (dvdsk) [inline templates/index.html:242]: why is this commented?

Comment (dvdsk) [reply templates/index.html:242]: same question regarding the other commented bits

Comment (luk1337) [reply templates/index.html:242]: it probably breaks JS parsing in code editors otherwise, and it's handled by the template system anyway

Comment (dvdsk) [reply templates/index.html:242]: ahh thanks (and thanks for the quick response)! I did not know that, I thought they where leftover commented bits from testing.

**Pull request merged**